linera-client/src/client_options.rs

```diff
@@ -96,6 +102,10 @@ pub struct ClientOptions {
     #[arg(long)]
     pub wasm_runtime: Option<WasmRuntime>,
 
+    /// The maximal number of simultaneous chains in memory handling requests.
+    #[arg(long, default_value = "20")]
```

Comment:
Should the default be higher?

Reply:
For the client? I'm not sure it's needed :man_shrugging:

Reply:
@afck Do we still need to push this number?

Reply:
Intuitively I would say yes, but last time I tried it it actually didn't make much of a difference. So I guess not for now.